ppr-ui/src/components/unitNotes/UnitNoteHeaderInfo.vue

```diff
@@ -1,8 +1,11 @@
 <template>
+  <div>
   <v-row no-gutters>
     <v-col cols="12" class="py-3">
       <h3> {{ noteHeader }} </h3>
     </v-col>
+  </v-row>
+  <v-row no-gutters>
```

Comment:
Separates into two rows to deal with width being too large causing the button to be in two lines